ccdproc/image_collection.py

```diff
@@ -216,6 +222,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str or int, The extension from which the header and data will be read in all files.
```

Comment:
It does work, but only if the str and the int value correspond to the same extension.
and only in the order `(str, int)`